packages/router-bridge/src/sagaRouter/module.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import { Router } from 'react-router-dom';
+
+import { connectRouter, routerMiddleware, ConnectedRouter } from 'connected-react-router';
```

Comment:
This file is imported in `src/sagaRouter/index.ts` which is imported in `src/index.ts`. But `connected-react-router` is an optional dep because this package is supposed to be compatible with RR v3.

This import from connected-react-router will fail in projects that don't depend on it. The error will be thrown when importing anything from router-bridge

But in fact this `sagaRouter` module is more a `router` module as it setups the router in the module's RootComponent and starts the saga router.
It is supposed to replace cmf-router that wraps RR v3. It should be in another dedicated place like cmf-router is.